src-script/build-release-package.js

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+/**
+ *
+ *    Copyright (c) 2023 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+const scriptUtil = require('./script-util.js')
+const os = require('os')
+const path = require('path')
+const fs = require('fs')
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
```

Comment:
What does hideBin do?

Reply:
It just helper that simplifies command line parsing